extensions/ql-vscode/package.json

```diff
@@ -461,13 +461,17 @@
         "command": "codeQLQueryHistory.showQueryText",
         "title": "Show Query Text"
       },
+      {
+        "command": "codeQLQueryHistory.exportCsvResults",
+        "title": "Export Results (CSV)"
+      },
       {
         "command": "codeQLQueryHistory.viewCsvResults",
-        "title": "View Results (CSV)"
+        "title": "View Alerts (CSV)"
       },
       {
         "command": "codeQLQueryHistory.viewSarifResults",
-        "title": "View Results (SARIF)"
+        "title": "View Alerts (SARIF)"
```

Comment:
> `@github/docs-content-codeql` has been cc'd in all issues for UI or other user-facing changes made by this pull request.

Slightly tangential, but the docs are a bit outdated in this area and still mention a **View SARIF** command, instead of **View Results/Alerts (SARIF)**. Looks like this section (https://github.com/github/codeql/blob/main/docs/codeql/codeql-for-visual-studio-code/analyzing-your-projects.rst#viewing-query-results) should mention the correct name for this command, as well as mentioning CSV. I'll open a quick PR for this (and avoid pinging in the docs team) 🙂

Reply:
Awesome, thanks for volunteering to do that 🙂